backend/src/models/comment.js

```diff
@@ -63,7 +63,7 @@ class Comment {
             {
                 [COMMENTS]: { [key]: value }
             })
-            // TODO: check comment exists and 404 if not
+            // TODO: check parent object exists and 404 if not
```

Comment:
400 if not ?

Reply:
I'm not sure - I thought 404 because you're trying to (e.g.) POST an answer to a question that doesn't exist.